lib/editors.coffee

```diff
@@ -55,7 +55,12 @@ class QuillJs extends Editor
   observeLocalText: (backend)->
     @editor.on "text-change", (deltas, source)->
       # call the backend with deltas
-      backend deltas.ops
+      position = backend deltas.ops
+      # trigger an extra event to move cursor to position of inserted text
+      @editor.selection.emitter.emit(
```

Comment:
I don't think that you are supposed to do that.
1. `.emit` should only be called by quill
2. This is quill related. So this does not belong to y-richtext.coffee (put it in Editors)

Isn't there another way?